src/test/java/com/google/api/codegen/testdata/discoveries/go/go_logging.v2beta1.json.baseline

```diff
@@ -22,25 +25,33 @@ import (
 
 func main() {
   ctx := context.Background()
+
+  // Authentication is provided by the `gcloud` tool when running locally
```

Comment:
@broady 

I saw you had a comment about this in the other PR, did you still want it changed to something else?